src/ServiceControl.Transports/IThroughputQuery.cs

```diff
@@ -35,11 +37,11 @@ public class QueueThroughput
 {
     public DateOnly DateUTC { get; set; }
     public long TotalThroughput { get; set; }
-    public string? Scope { get; set; }
-    public string[] EndpointIndicators { get; set; } = [];
 }
 
 public interface IQueueName
```

Comment:
I think this should be renamed to IQueueDetails or something - it's not just the name any more

Reply:
good point

Reply:
Done